ConnectionManager.java

```
@@ -0,0 +1,3 @@
+import java.sql.*;
+public class ConnectionManager{
+    public static Connection getConnection(){try{Connection con=DriverManager.getConnection("jdbc:mysql://localhost:3306/test","root","root");return con;}catch(Exception e){}return null;}}
```

The method `getConnection` should have proper exception handling. Silently catching exceptions without logging or rethrowing them can hide bugs.